server/routerlicious/packages/services-utils/src/test/throttlerMiddleware.spec.ts

```diff
@@ -1,135 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import assert from "assert";
-import express from "express";
-import request from "supertest";
-import { IThrottler } from "@fluidframework/server-services-core";
-import { TestThrottler } from "@fluidframework/server-test-utils";
-import { throttle, IThrottleMiddlewareOptions } from "../throttlerMiddleware";
-
```

Comment:
Why are we deleting this file?

Reply:
I didn't mean to change service-utils. I reverted those changes. Thanks